PreguntasFrecuentes

MareyVS opened the pull request:
Se incluye pantalla para preguntas frecuentes

crc08 committed:
Se sube branch con el proposito de revisar codigo implementado para la pantalla de preguntas precuentes

MareyVS committed:
Se modifica lìnea 14

còdigo correcto: .env.alpha

MareyVS committed:
variable name change index -> Index

tomaslosada reviewed (dismissed):
Approved

MareyVS committed:
Se atienden "code smell" issues de sonarcloud

crc08 committed:
se actualiza archivo: src/presentation/resources para correcto formato de entrada de texto

fedelmar commented on src/src/presentation/resources/commonQuestionsCatalog.ts:
```diff
@@ -0,0 +1,245 @@
+            ' Se puede utilizar la barra para filtrar por categorías según la necesidad. Las categorías fueron establecidas por Semillas, son las mismas que se usan para ' +
+            ' categorizar sus créditos. ' +
+            ' Una vez seleccionado el prestador que se requiere, se va a consultar por quién va a usar el beneficio. Puede ser para el titular o para alguien del grupo ' +
+            ' familiar declarado en Semillas. Presionar Siguiente para mostrar la información que va a compartir la app. '
```
Falta un + entre los siguientes string

MareyVS committed:
Se agrega '+' en línea 224

MareyVS committed:
OpenEmail: se corrige variable

MareyVS committed:
se modifica formato de letra y colores

MareyVS committed:
Se agregan variables a la clase OpenEmailScreen

crc08 committed:
Se integran últimos cambios relacionados a la funcionalidad --PreguntasFrecuentes--

crc08 committed:
Merge branch 'preguntasFrecuentes' of https://github.com/ong-bitcoin-argentina/DIDI-SSI-Mobile into preguntasFrecuentes

MareyVS committed:
Merge branch 'develop' into preguntasFrecuentes

MareyVS committed:
linea 303 - se agrega etiqueta -- //sensitive --

sonarcloud issue

MareyVS committed:
se eliminan lineas duplicadas - sonarcloud issue

MareyVS commented:
![PantallaPF-1](https://user-images.githubusercontent.com/95529836/163840658-f323b37c-c2d8-42dc-89ff-286067a45f6f.jpeg)
![PantallaPF-2](https://user-images.githubusercontent.com/95529836/163840665-a208c798-266e-4e5e-8e11-84a5de8cc182.jpeg)
![preguntasMasFrecuentes](https://user-images.githubusercontent.com/95529836/168424957-d4ffe24e-46a0-4df6-9e61-b4a46d85a4ad.jpeg)
![PantallaPF-4](https://user-images.githubusercontent.com/95529836/163840672-0a76b622-ba8e-44fc-abfd-cdd0805f6b36.jpeg)

MatiasOS commented:
> ![PantallaPF-1](https://user-images.githubusercontent.com/95529836/163840658-f323b37c-c2d8-42dc-89ff-286067a45f6f.jpeg) ![PantallaPF-2](https://user-images.githubusercontent.com/95529836/163840665-a208c798-266e-4e5e-8e11-84a5de8cc182.jpeg) ![PantallaPF-3](https://user-images.githubusercontent.com/95529836/163840670-d5d1c780-9c4a-4bd3-8c37-ee9f5f16df59.jpeg) ![PantallaPF-4](https://user-images.githubusercontent.com/95529836/163840672-0a76b622-ba8e-44fc-abfd-cdd0805f6b36.jpeg)

@mscanepa qué te parece el **?** en la ui? Habría que probar con un celu de baja resolución a ver como queda no?

mscanepa commented:
Me parece bien el lugar para el ícono, usaria uno redondo con el "?" adentro para que no quede tan suelto. 

La letra de la respuesta a la pregunta me resulta muy chica pero la cantidad de texto e informacion a presentar es mucha y no hay manera de que eso este en fuente grande, en realidad deberia ser menos texto para el celular pero no es algo critico por lo tanto no bloquearia mucho el avance de esto por ese detalle.

Todo bien por mí.

MareyVS committed:
Se habilita espacio de correo electrónico

MareyVS committed:
correo electronico

Se modifica archivo para implementar opción de correo electrónico

MareyVS committed:
correo electronico

Se agrega: 
Linea 19: import { OpenEmailScreen } from "../../common/OpenEmail";
linea 45: OpenEmail: NavMap.from(OpenEmailScreen,{}),

MareyVS committed:
correo electrónico

Se agrega:
linea 26: import { OpenEmailScreenProps } from "../../common/OpenEmail";
linea 52: OpenEmailScreenProps: OpenEmailScreenProps

MareyVS committed:
correo electronico

linea: 127: "contact". Se agrega String correspondiente a correo electrónico

MareyVS committed:
Merge branch 'develop' into preguntasFrecuentes

MareyVS committed:
Se solucionan errores de SonarCloud

MareyVS committed:
se resulven observaciones de LGTM

se elimina variable no usada

MareyVS committed:
Se modifica dirección de correo electrónico

líneas 10 y 11

MareyVS committed:
Merge branch 'develop' into preguntasFrecuentes

MareyVS committed:
Se cambia concepto de ASUNTO 

Linea 38 -> SendMail -> SOPORTE DIDI

MareyVS committed:
se elimina archivo autogenerado

MareyVS committed:
mailBody

Before -> "Información"
mailBody = "Escribe tu consulta"

MareyVS commented:
![abrirEmail](https://user-images.githubusercontent.com/95529836/168426999-f4714141-2587-493f-bdc4-ceb1dbdb111a.jpeg)
![ComposeEmail](https://user-images.githubusercontent.com/95529836/168426996-43276cef-5f48-47ac-8783-9eff474a7f3c.jpeg)

mscanepa commented:
El email de soporte de aidi es soporte@aidi.app

farpires commented on src/src/presentation/common/CommonQuestions.tsx:
```diff
@@ -0,0 +1,100 @@
+							<CollapseBody key={index} >
+								{ questions[key].map((question, Index) => {
+									return(
+										<Collapse style={{ paddingHorizontal: 0 }}>
```
agregar key prop : <Collapse key={Index}

farpires commented on src/src/presentation/common/OpenEmail.tsx:
```diff
@@ -0,0 +1,52 @@
+import { StyleSheet, Linking } from 'react-native';
+import colors from "../resources/colors";
+import strings from "../resources/strings";
+const mailTo = "info@didi.org.ar";
```
cambiar email a: soporte@aidi.app

farpires commented on src/src/presentation/resources/strings.ts:
```diff
@@ -1,1051 +1,1011 @@
+		}
+	},
+	contact: {
+		title: "Si las preguntas frecuentes no han resuelto tu duda, envía un mensaje al correo electrónico info@didi.org.ar",
```
cambiar email a: soporte@aidi.app

farpires reviewed (commented):
al finalizar resolver conflicto ( traer los cambios del branch develop  a preguntasFrecuentes )
y probar su funcionamiento.

farpires commented on src/src/presentation/common/EnterEmail.tsx:
```diff
@@ -42,8 +42,10 @@ export class EnterEmailScreen extends React.Component<AddChildren<EnterEmailProp
+		
 		return (
 			<DidiScreen>
+				<DidiText.Explanation.Emphasis>la wea fome</DidiText.Explanation.Emphasis>
```
al actualizar con develop este comentario se cambiaría
